.github/actions/kibana-ftr/action.yml

```diff
@@ -0,0 +1,79 @@
+name: 'Run Kibana FTR'
+description: 'Run Kibana tests using FTR'
+
+inputs:
+  test_kibana_url:
+    description: 'URL for the Kibana instance to test'
+    required: true
+  test_es_url:
+    description: 'URL for the ES instance'
+    required: true
+  es_version:
+    description: 'Version of Elasticsearch to test against'
+    required: true
+  ref_commit_sha:
+    description: 'Kibana PR commit sha'
+    required: false
+    default: 'main'
+  test_browser_headless:
+    description: 'Whether to run tests in headless mode'
+    required: false
+    default: '1'
+  test_cloud:
```

Comment:
According to the [FTR documentation](https://www.elastic.co/guide/en/kibana/current/development-tests.html#_running_functional_tests), when running against an Elastic Cloud instance, additional environment variables are required: `TEST_CLOUD` and `ES_SECURITY_ENABLED`. The `input.test_cloud` environment variable should be configured for our case. The `input.config` variable instructs FTR which test configuration to execute. Currently, it will only execute `config.cloud.ts`, so we can omit both input variables for now and extend to work with different test configurations later.